examples/block_author.ts

```diff
@@ -1,8 +1,6 @@
-import { consensus } from "capi/patterns"
```

Comment:
Given that we may reference chain-specific capi server routes from this patterns dir, we don't want to expose everything from the root (as it could spin up irrelevant test nodes).